### PR TITLE
fix(readme): start/stop commands & more doc

### PR DESCRIPTION
**Pull request by jdreo**
- Fix the start/stop commands that were missing the "server" keyword.
- Add a note about JAVA_HOME in the Ubuntu section.
- Add a note for when to try Maven memory increase.

### CODE REVIEW

**Comment by mnd999** [comment]
Hello and thank you for the PR!

In order for us to accept it, you have to sign our CLA, more information is here: https://neo4j.com/developer/cla/

**Comment by jdreo** [comment]
Done, thanks for considering this contribution.

**Comment by klaren** [comment]
The CLA is received and verified. I'll trigger the integration.

This will be included in the 5.13 release in October. Thank you once again for the PR :smile:

**Review by klaren — APPROVED** [review]

File: README.asciidoc [hunk 1]
```diff
@@ -32,25 +32,28 @@ Please note that we do not support building Debian packages on macOS.
...
 * A plain `mvn clean install -T1C` will only build the individual jar files.
 * Test execution is, of course, part of the build.
 * In case you just want the jars, without running tests, this is for you: `mvn clean install -DskipTests -T1C`.
-* You may need to increase the memory available to Maven: `export MAVEN_OPTS="-Xmx2048m"`.
+* You may need to increase the memory available to Maven: `export MAVEN_OPTS="-Xmx2048m"` (try this first if you get build errors).
 * You may run into problems resolving `org.neo4j.build:build-resources` due to a bug in maven. To resolve this simply invoke `mvn clean install -pl build-resources`.
 
 == Running Neo4j ==
 
 After running a `mvn clean install`, `cd` into `packaging/standalone/target` and extract the version you want, then:
```
Comment (usererror838) [inline README.asciidoc:50]: 'mvn clean install' , 'JKT795781'

**Pull request closed without merging**